README.md

```diff
@@ -157,6 +157,17 @@ The advanced properties that can be configured for the connector are:
 | client.use.binary.protocol | Set to `false` to force the library to use JSON encoding for REST and realtime operations, instead of msgpack encoding. | *Boolean* | True |
 | client.loglevel | Sets the verbosity of logging. | *Integer* | 0 |
 
+## Dynamic Channel Configuration
```

Comment:
I think the description of the `channel` configuration option in the above table should like to this section.

Reply:
It took a couple of attempts to get this right :) See
https://github.com/ably/kafka-connect-ably/pull/45/commits/5ccf1de99c4ef3cc5eedc9d1cda955ab3b82b6dd
https://github.com/ably/kafka-connect-ably/pull/45/commits/e423a4ff1c767cc1f9ca7628502c6dfd294d4dc1
https://github.com/ably/kafka-connect-ably/pull/45/commits/002669cdf4b8a25def9aee20bb6e3d3a4c713b9d